FOSSAsia/DecisionViewController.swift

```diff
@@ -0,0 +1,25 @@
+import UIKit
+
+class DecisionViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        if UserDefaults.exists(key: Constants.UserDefaultsKey.acessToken) {
```

Comment:
Put code in a function and call that function here

Reply:
okk @imjog !!